<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccuracyValidationToolClass</class>
 <widget class="QMainWindow" name="AccuracyValidationToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AccuracyValidationTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>12</number>
    </property>
    <item row="3" column="0" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_frameRate">
          <property name="text">
           <string> Frame Rate:     </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_frameRate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_frameRate">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_recordInterval">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> Report Interval:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_recordInterval">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_recordInterval">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_startTime">
          <property name="text">
           <string> Start Time:     </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit_startTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="displayFormat">
           <string>yyyy/MM/dd HH:mm:ss</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_startTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="7">
     <widget class="QComboBox" name="comboBox_speed">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>x0.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x1.0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x2.0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x4.0</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0" colspan="8">
     <widget class="QLabel" name="label_video">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3" rowspan="3" colspan="5">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_countIn">
          <property name="text">
           <string> Count of In: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_countIn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_inMinusOne">
          <property name="text">
           <string>      -1       </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_countOut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> Count of Out:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_countOut">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_outMinusOne">
          <property name="text">
           <string>      -1       </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_open">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> Open a Video  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_export">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>  Export File  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close">
          <property name="text">
           <string>  Close Video  </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="accuracyvalidationtool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_open</sender>
   <signal>clicked()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_openAVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>692</y>
    </hint>
    <hint type="destinationlabel">
     <x>791</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_speed</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_videoSpeed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>945</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>924</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_HSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_HSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_frameRate</sender>
   <signal>clicked()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_setFrameRate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_recordInterval</sender>
   <signal>clicked()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_setRecordInterval()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>648</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_startTime</sender>
   <signal>clicked()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_setStartTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>692</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>685</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_inMinusOne</sender>
   <signal>clicked()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_countInMinus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>883</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>840</x>
     <y>603</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_outMinusOne</sender>
   <signal>clicked()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_countOutMinus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>943</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>841</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_export</sender>
   <signal>clicked()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_exportFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>692</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>AccuracyValidationToolClass</receiver>
   <slot>slot_closeVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>877</x>
     <y>681</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>708</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_openAVideo()</slot>
  <slot>slot_HSliderMoved(int)</slot>
  <slot>slot_videoSpeed(int)</slot>
  <slot>slot_HSliderReleased()</slot>
  <slot>slot_setFrameRate()</slot>
  <slot>slot_setRecordInterval()</slot>
  <slot>slot_setStartTime()</slot>
  <slot>slot_countInMinus()</slot>
  <slot>slot_countOutMinus()</slot>
  <slot>slot_exportFile()</slot>
  <slot>slot_closeVideo()</slot>
 </slots>
</ui>
